style/icons/shield40_us_bia_3.svg

```diff
@@ -0,0 +1,4 @@
+<svg height="20" width="26" xmlns="http://www.w3.org/2000/svg">
+ <path style="fill:#000" d="M 4.5,0 C 3.9477381,5.5217905e-5 3.5000552,0.44773812 3.5,1 v 0.6894531 c 0.036268,0.070803 0.080854,0.1370266 0.132813,0.1972656 l -0.466797,-0.1875 C 2.6308551,1.4856636 2.0264032,1.7673238 1.845703,2.3144531 L 0.408203,6.6875 C 0.3747909,6.7883097 0.35764916,6.8937978 0.357422,7 c 4.3e-5,6.112868 5.505377,11.092155 12.378906,12.964844 0.08631,0.02359 0.17499,0.03538 0.263672,0.03538 0.08868,0 0.177364,-0.01179 0.263672,-0.03538 C 20.137201,18.092155 25.642535,13.112868 25.642578,7 25.642351,6.8937979 25.62521,6.7883097 25.591798,6.6875 l -1.4375,-4.3730469 C 23.973598,1.7673233 23.369145,1.4856629 22.833984,1.6992187 l -0.466797,0.1875 C 22.419146,1.8264797 22.463732,1.7602565 22.5,1.6894531 V 1 C 22.499945,0.44773812 22.052262,5.5217905e-5 21.5,0 Z"/>
```

Comment:
At a glance, this shield looks a little more like a corner-notched projectile point than the polished vessel I came up with for Rebusurance, but it’s still rather blunt. How about allowing the text to shrink a bit more, easing the pressure to widen the shield, as in #234? Angling the notches downward could help too.

Reply:
Here's the thin variant only with no other changes. It looks fine for up to 3 digits, though 4-digit routes look crowded. However, I'm not convinced that any 4-digit routes are actually posted with these shields, so I guess it's a moot point.
![Screenshot from 2022-05-31 08-42-50](https://user-images.githubusercontent.com/1732117/171176108-6adc4f2e-112c-4154-9f13-0a6deed910bb.png) ![Screenshot from 2022-05-31 08-43-00](https://user-images.githubusercontent.com/1732117/171176111-34d124a3-2ac7-4690-af28-5523fbe5445c.png)
![Screenshot from 2022-05-31 08-43-37](https://user-images.githubusercontent.com/1732117/171176113-c5f51b52-0891-4084-bcbe-6331779c788c.png) ![Screenshot from 2022-05-31 08-43-53](https://user-images.githubusercontent.com/1732117/171176115-a560efa6-65bf-415c-950b-abb61ed2a655.png)

Reply:
And with the notch angled downward a little more:
![Screenshot from 2022-05-31 09-06-15](https://user-images.githubusercontent.com/1732117/171180617-cca0fd82-4ffc-4925-aa3a-36436adc018d.png) ![Screenshot from 2022-05-31 09-06-41](https://user-images.githubusercontent.com/1732117/171180624-9343e631-0fe8-4810-9583-b36b4275f43d.png)
![Screenshot from 2022-05-31 09-06-28](https://user-images.githubusercontent.com/1732117/171180622-569beb8e-926c-4730-b667-11871730fa12.png) ![Screenshot from 2022-05-31 09-06-50](https://user-images.githubusercontent.com/1732117/171180625-79362cf9-51de-4df4-b180-c05f174757af.png)